docs/src/radiation.md

```diff
@@ -58,19 +59,140 @@ flux from below at interface ``k+1/2`` (``k`` increases downwards, see
 ``\Delta p = p_{k+1/2} - p_{k-1/2}`` is the pressure thickness of layer ``k``,
 gravity ``g`` and heat capacity ``c_p``.
 
-
 ## Shortwave radiation
 
-Currently implemented is
+Currently implemented schemes:
 
 ```@example radiation
+using SpeedyWeather
 subtypes(SpeedyWeather.AbstractShortwave)
 ```
 
+## OneBandShortwave: Single-band shortwave radiation with diagnostic clouds
+
+*Note: Stratocumulus cloud parameterization and ozone-radiation interactions are currently not implemented in the `OneBandShortwave` scheme.*
+
+The `OneBandShortwave` scheme provides a single-band (broadband) shortwave radiation parameterization, including diagnostic cloud effects following [^KMB06].
+
+**Cloud diagnosis:**
+Cloud properties are diagnosed from the relative humidity and total precipitation in the atmospheric column. The cloud base is set at the interface between the lowest two model layers, and the cloud top is the highest layer where both
+
+$$
+\mathrm{RH}_k > \mathrm{RH}_{cl} \quad \text{and} \quad Q_k > Q_{cl}
+$$
+
+are satisfied. The cloud cover (CLC) in a layer is then given by
```

Comment:
I know this is written in the Fortran SPEEDY documentation but this is, as it's currently implemented actually not true. `cloud_top` starts at `nlayers+1` then if large-scale condensation occurs it'll be moved up to the highest layer k where condensation occurs `cloud_top = min(k, cloud_top)` and similar for convection. You could see what difference it makes to actually implement the `cloud_top` calculation as outlined here